snowflake-py-pulumi-search-export/Pulumi.yaml

```diff
@@ -0,0 +1,6 @@
+name: snowflake-pulumi-search-export-py
+runtime:
+  name: python
+  options:
+    virtualenv: venv
+description: A minimal Python Pulumi program
```

Comment:
Should this be something like "An example of exporting Pulumi Cloud data to Snowflake"?